test/gtest/ucp/test_ucp_am.cc

```diff
@@ -965,9 +975,7 @@ class test_ucp_am_nbx_eager_data_release : public test_ucp_am_nbx {
 
     static void get_test_variants_reply(std::vector<ucp_test_variant> &variants)
     {
-        add_variant_values(variants, test_ucp_am_base::get_test_variants, 0);
-        add_variant_values(variants, test_ucp_am_base::get_test_variants,
-                           UCP_AM_SEND_FLAG_REPLY, "reply");
+        test_ucp_am_nbx_reply::get_test_variants(variants);
```

Comment:
maybe remove get_test_variants_reply wrapper, and use test_ucp_am_nbx_reply::get_test_variants directly from  test_ucp_am_nbx_eager_data_release::get_test_variants